### PR TITLE
chore: fix mux-audio and mux-video examples

### PR DESCRIPTION
**Pull request by dylanjha**

### CODE REVIEW

**Review by gkatsev — APPROVED** [review]

File: README_INTERNAL.md [hunk 1]
```diff
@@ -20,10 +20,16 @@ and Github Actions for continuously publishing Mux elements.
 
 > Make sure you have a `GH_TOKEN` environment variable set to [create the release notes](https://github.com/lerna/lerna/tree/main/commands/version#--create-release-type) in Github.  
 > This token can be created in Github under [Settings > Developer settings > Personal access tokens](https://github.com/settings/tokens).
+> The token you create should have all the `repo` scope permissions.
+> After creating the token select "Configure SSO" and authorize Mux Inc.
 
-1. From the workspace root directory, run `yarn version:update`.
+1. `cd` into the `elements` directory where the origin is set to `muxinc/elements`. Don't use your forked repo, use a separate directory that is cloned from `https://github.com/muxinc/elements`.
```
Comment (cjpillsbury) [inline README_INTERNAL.md:26]: nit (non-blocking): Copy edit -
Comma should be semicolon or a new sentence.

File: README_INTERNAL.md [hunk 2]
```diff
@@ -20,10 +20,16 @@ and Github Actions for continuously publishing Mux elements.
...
+1. From the workspace root directory, run `yarn install`.
+1. From the workspace root directory, run `GH_TOKEN=xxxx yarn version:update`.
    This will bump the version of the packages changed since the last release and push those changes to Github.
    You might have to manually correct the version if the suggestion of Conventional Commits is not right.
+1. Confirm in the [Releases tab](https://github.com/muxinc/elements/releases) that the releases has been created.
 1. Go to the [Github Actions tab for elements](https://github.com/muxinc/elements/actions), select "Deployment" action in the left sidebar
 1. Click the "Run workflow" dropdown and "Run workflow" on the `main` branch
-1. Starting this workflow will require an approval. After approving this will kick off the deploy action which will publish all the updated packages to NPM
+1. Starting this workflow will require an approval from a core contributor. Click into the 'pending' workflow to get it approved.
+1. After approving this will kick off the deploy action which will publish all the updated packages to NPM
```
Comment (cjpillsbury) [inline README_INTERNAL.md:35]: nit (non-blocking). Add something like this after "After approving":

> Confirm the package(s) were successfully published to npm by checking the latest version and publish time npmjs, e.g. https://www.npmjs.com/package/@mux-elements/mux-player

**Review by cjpillsbury — CHANGES_REQUESTED** [review]
One minor blocking ask and a couple of non-blocking ones. Otherwise, 😎

**Review by cjpillsbury — APPROVED** [review]
🏄‍♂️

**Pull request merged**